CHANGELOG.md

```diff
@@ -80,6 +83,16 @@ A brief description of the categories of changes:
   invalid usage previously but we were not failing the build. From now on this
   is explicitly disallowed.
 * (toolchains) Added riscv64 platform definition for python toolchains.
+* (rules) A new bootstrap implementation that doesn't require a system Python
+  is available. It can be enabled by setting
+  {obj}`--@rules_python//python:config_settings:bootstrap_impl=two_phase`. It
```

Comment:
It looks a typo, should be 

`--@rules_python//python/config_settings:bootstrap_impl`

Reply:
Oops, good catch, thanks @hunshcn 

Sent https://github.com/bazelbuild/rules_python/pull/1959